clouddrift/wavelet.py

```diff
@@ -0,0 +1,518 @@
+"""
+This module provides wavelet functions.
+"""
+
+import numpy as np
+from typing import Optional, Tuple, Union
+import xarray as xr
+import warnings
+from scipy.special import gamma as _gamma, gammaln as _lgamma
+
+
+def wavetrans(
+    x: np.ndarray,
+    wave: np.ndarray,
```

Comment:
Would it make sense to call this `wavelets`? `wave` is very general, I think.

Reply:
changed to `wavelet`